platform-sdk/settings.gradle.kts

```diff
@@ -42,7 +44,7 @@ include(":swirlds-fcqueue")
 
 include(":swirlds-merkle")
 
-include(":swirlds-jasperdb")
+include(":swirlds-merkledb", "swirlds-jasperdb")
```

Comment:
Are we renaming merkledb to jasperdb here?

Reply:
this definitely should not be the case, as JasperDB is now removed from the codebase